_gsocproposals/2020/proposal_IntAlertSystem.md

```diff
@@ -0,0 +1,58 @@
+---
+title: Intelligent Alert Management System For HEP experiments
+layout: gsoc_proposal
+project: Monitoring
+year: 2020
+organization: CERN
+---
+
+## Description
+
+We would like to build an intelligent Alert Management System for HEP experiments.
+It should embrace existing open-source solutions based on Prometheus and AlertManager
+to select alerts from various sub-systems and reduce amount
+of non-relevant alerts. The system should be able to correlate alerts from
+CERN MONIT infrastructure based on ElasticSearch, Kafka, Grafana stack, GGUS WLCG, and
+Prometheus, AlertManager, VictoriaMetrics software components used in HEP experiments.
+
+The system should collect various alerts from ES, InfluxDB, Prometheus data-sources,
+GGUS system and be able to aggregate, and create meaningful alerts to address
+various sub-system failures. For instance, if GRID site has experience a problem
```

Comment:
"if some GRID site has experienced a problem ...". Also, students may not be aware of what GRID is, nor various names of subsystems you enumerate before. I would rephrase in a more general way of give a minimal explanation for these terms. Remember the targed is students un-aware of HEP.experiment jargon.